components/CommentCard.jsx

```diff
@@ -1,14 +1,27 @@
+import { useState } from "react"
+
 export default function CommentCard({ comment }) {
+    const [votes, setVotes ] = useState(comment.votes)
```

Comment:
votes is already stored in state, with your comments data - if the comments update, this component will automatically rerender. Therefore passing it into state here duplicates the data that is stored and doesn't help the functionality of the app.